resources/views/components/card/address.blade.php

```diff
@@ -0,0 +1,37 @@
+@props(['shipping' => false, 'billing' => false, 'dynamicType' => false, 'check' => false])
+
+<address-card
+    {{ $attributes->whereStartsWith('v-') }}
+    @if(!$dynamicType)
+        :shipping="{{ var_export($shipping) }}"
+        :billing="{{ var_export($billing) }}"
+    @endif
+    v-slot="{ address, billing, shipping, isEmpty }"
+>
+    <div {{ $attributes->whereDoesntStartWith('v-')->class('relative bg-white border px-8 py-6 rounded max-w-[400px]') }} v-if="!isEmpty">
+        <div class="h-full flex flex-col">
+            @if($check)
+                <template v-if="{{ $check }}">
+                    <div class="absolute w-1 left-0 inset-y-0 rounded-l bg-accent"></div>
+                    <x-heroicon-s-check class="absolute w-5 right-7 top-7 text-accent"/>
+                </template>
+            @endif
+            <x-rapidez-ct::title.lg class="mb-4 pr-8">
+                <template v-if="billing && shipping">@lang('Shipping & billing address')</template>
+                <template v-else-if="shipping">@lang('Shipping address')</template>
+                <template v-else-if="billing">@lang('Billing address')</template>
+                <template v-else>@lang('Address')</template>
+            </x-rapidez-ct::title.lg>
+            <div class="flex flex-wrap justify-between h-full">
```

Comment:
does h-full work in safari? have you checked? you can probably fix this with a flex and flex-1

Reply:
Works just fine on my machine :)